src/pages/PostPages/StepIndicator/style.css.ts

```diff
@@ -0,0 +1,31 @@
+import { css, cva } from "@styled-system/css";
+
+export const progressCircle = cva({
+    base: {
+        height: '0.375rem',
+        transition: 'all 0.3s ease-in-out'
+    },
+    variants: {
+        color: {
+            main: {
+                // width: '0.875rem',
+                backgroundColor: 'main',
+                borderRadius: '0.625rem',
+            },
+            gray: {
```

Comment:
```suggestion
        isActive: {
            true: {
                backgroundColor: 'main',
                borderRadius: '0.625rem',
            },
            false: {
```
이런 식으로 하고, StepIndicator 컴포넌트에서 그냥 이런식으로 넣어주면 더 깔꼼할거 같애요
```ts
	const isActive = stepNumber <= currentStep ? 'main' : 'gray';
	// ...
	className={s.progressCircle({ isActive })}
```